src/tasks/transferEventLogs/db/block.crud.ts

```diff
@@ -0,0 +1,35 @@
+import { prismaClient } from "db-client";
+
+export async function getBlockNoOnDatabase() {
```

Comment:
probably OnDatabase stutters in names in general, the methods are already scoped in the db folder